perun-core/src/main/java/cz/metacentrum/perun/core/impl/FacilitiesManagerImpl.java

```diff
@@ -511,8 +534,9 @@ public void removeOwner(PerunSession sess, Facility facility, Owner owner) throw
 
 			} else if (specificService != null) {
 
-				return jdbc.query("select " + ResourcesManagerImpl.resourceMappingSelectQuery + " from resource_services join resources on " +
-						"resource_services.resource_id=resources.id where facility_id=? and service_id=?",
+				return jdbc.query("select " + ResourcesManagerImpl.resourceMappingSelectQuery + " from resource_services" +
```

Comment:
The formating of the SQL commands in the whole PR is much better than the previous, but I have some comments which you may consider if you like.

1. It would be better to make the indentation the same in each SQL command. For example, some of the indents after the first line have two extra tabs and some don't. Or some of the mappers start on a separate line and some don't.
2. I would consider putting the "from"  clause on the separate line, because of the readability.
3. Maybe even putting the "select" clause on the second line would help to align the indentation better, but I am not sure in this one.

Reply:
Thanks for the suggestions. We should agree on some kind of standard within our code for SQL queries in the core team. I would prefer cleaning up those SQL afterwards.

I just want really simple queries to stay simple (eg. single line) and not to take 4-5 lines. For complex SQLs, more lines is better. I'm in favor of having "from" at the newline to.

Moving "select" to the next line causes just more indentation to apply. We generally do not have newlines between braces.